index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Comic+Neue:wght@300;400;700&family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./index.css">
+  <title>AwesomeBooks</title>
+</head>
+<body>
+  <header class ="navBar">
+    <!--Navigation bar-->
+    <div class="nbTitle">Awesome Books</div>
+    <nav>
+      <ul class="bookList-add-contact">
+          <li class="navList">List</li>
+          <span></span>
+          <li class="navAdd">Add new</li>
+          <span></span>
+          <li class="navCont">Contact</li>
+      </ul>
+    </nav>
+  
+  </header>
+  <p id="datetime"></p>
+  <!--1rst page-->
+  <!--Added Books-->
+  <main>
+    <div class="list">
+    <h1>Awesome Books</h1>
+    <div class="book-list-container">
+      <h2 class="subtitle">All Awesome Books</h2>
+      <!--JavaScript Dynamic Book List-->
+      <ul id="bookList"></ul>
+    </div>
+  </div>
+  </main>
+  <!--Add a book form-->
+  <div class="form-container hide">
+    <h2 class="subtitle">Add a Book</h2>
+      <form class= "add-form" id="add-form">
+        <input type="text" required placeholder="Title of the book" id="title">
+        <input type="text" required placeholder="Author of the book" id="author">
+        <button type="submit" id="submit">Add</button>
+      </form>
```

Comment:
- [ ] [Optional] You should consider adding some kind of alert to notify the user about the action.